web/client/actions/mediaEditor.js

```diff
@@ -66,26 +78,21 @@ export const saveMediaSuccess = ({ mediaType, source, data, id }) => ({ type: SA
  * @param {string} id of the resource
  */
 export const selectItem = (id) => ({ type: SELECT_ITEM, id});
-// RESOURCE FORMAT DRAFT :
-/*
-{
-    type: 'image'|'video'|'map'|'iframe'|'document' // (pdf)
-    source: 'id' // id of the source, just to identify it in a local context
-    data: {
-        //specific data for the source type
-    }
-}
-*/
+/**
+ * update item in media editor list
+ * @param {object} param.item
+ */
+export const updateItem = ({map}) => ({ type: UPDATE_ITEM, item: map});
```

Comment:
this map object is unnecessary in the item
```suggestion
export const updateItem = (item => ({ type: UPDATE_ITEM, item});
```